tekton/pipeline/sanity-teste-pipeline.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: sanity-test-pipeline
```

Comment:
nit: the file name and the pipeline name don't match

Reply:
Thank you! Let me update it.